src/containers/Statistics/Statistics.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { StatsCollateralChart } from './../../components/Stats/StatsCollateralChart/StatsCollateral'
+import { DebtPoolContainer } from './../DebtPoolContainer/DebtPoolContainer'
+import { useSelector } from 'react-redux'
+import { getCollateralStructure, getSyntheticsStructure } from './../../store/selectors/exchange'
```

Comment:
In `tsconfig.paths.json` we have declare paths.
IMO this:
```
import { StatsCollateralChart } from '@components/Stats/StatsCollateralChart/StatsCollateral'
import { getCollateralStructure, getSyntheticsStructure } from '@selectors/exchange'
```
looks better than that:
```
import { StatsCollateralChart } from './../../components/Stats/StatsCollateralChart/StatsCollateral'
import { getCollateralStructure, getSyntheticsStructure } from './../../store/selectors/exchange'
```

Reply:
Done.